pkg/providers/nutanix/template.go

```diff
@@ -203,6 +204,12 @@ func buildTemplateMapCP(
 		"apiServerCertSANs":            clusterSpec.Cluster.Spec.ControlPlaneConfiguration.CertSANs,
 	}
 
+	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
+	if err != nil {
+		return nil, err
+	}
+	values["auditPolicy"] = auditPolicy
+
```

Comment:
nit: can u move the logic from line 207 before https://github.com/aws/eks-anywhere/blob/2e337b7956a1f673f83455725f7c2cc8d0ada50b/pkg/providers/nutanix/template.go#L162 so that u can add `auditPolicy` directly there with all the other default vars.

Reply:
Fixed